services/ui-src/src/forms/mcpar/mcpar.json

```
@@ -155,10 +155,25 @@
           "intro": {
             "section": "Section A: Program Information",
             "subsection": "Add BSS entities (A.8)",
+            "spreadsheet": "A_Program_Info",
             "info": [
               {
-                "type": "text",
-                "content": "Enter the names of Beneficiary Support System (BSS) entities that support enrollees in the program for which the state is reporting data."
+                "type": "html",
+                "as": "span",
+                "content": "Enter the names of Beneficiary Support System (BSS) entities that support enrollees in the program for which the state is reporting data. Learn more about BSS entities at "
+              },
+              {
+                "type": "externalLink",
+                "content": "42 CFR 438.71",
+                "props": {
+                  "href": "https://www.ecfr.gov/current/title-42/chapter-IV/subchapter-C/part-438/subpart-B/section-438.71",
+                  "target": "_blank"
+                }
+              },
+              {
+                "type": "html",
+                "as": "span",
+                "content": " See Glossary in Excel Workbook for the definition of BSS entities.<br/><br/>Examples of BSS entity types include a: State or Local Government Entity, Ombudsman Program, State Health Insurance Program (SHIP), Aging and Disability Resource Network (ADRN), Center for Indepedent Living (CIL), Legal Assistance Organization, Community-based Organization, Subcontractor, Enrollment Broker, Consultant, or Academic/Research Organization."
               }
```

Nit: You'll want a period at the beginning of this, so there's a period after the link. Also they'll want these to be paragraphs. Also the same note about `type` and `as` as shown above.
```suggestion
              {
                "type": "html",
                "content": "<span>. See Glossary in Excel Workbook for the definition of BSS entities.</span><p>Examples of BSS entity types include a: State or Local Government Entity, Ombudsman Program, State Health Insurance Program (SHIP), Aging and Disability Resource Network (ADRN), Center for Indepedent Living (CIL), Legal Assistance Organization, Community-based Organization, Subcontractor, Enrollment Broker, Consultant, or Academic/Research Organization.</p>"
              }
```